packages/tracing/src/browser/metrics.ts

```diff
@@ -143,6 +146,46 @@ export class MetricsInstrumentation {
       this._measurements['cls'] = { value: metric.value };
     });
   }
+  
+  /**
+   * Capture the information of the user agent.
+   */
+  private _trackNavigator(transaction: Transaction): void {
+    const navigator = global.navigator as null | (Navigator & NavigatorNetworkInformation & NavigatorDeviceMemory);
```

Comment:
If you'd use `if (!('navigator' in global))` would it still require adding information/device types? If so, we can leave it as is.

Reply:
Yep. TypeScript does not provide the built-in interfaces for `NavigatorNetworkInformation` and `NavigatorDeviceMemory`.